src/util/badWordsEn.js

```diff
@@ -1,52 +1,69 @@
-  'hardcoresex', 'hell', 'heshe', 'hoar', 'hoare', 'hoer', 'homo', 'hore', 'horniest', 'horny',
-  'hotsex', 'jack-off', 'jackoff', 'jap', 'jerk-off', 'jism', 'jiz', 'jizm', 'jizz', 'kawk', 'knob',
-  'knobead', 'knobed', 'knobend', 'knobhead', 'knobjocky', 'knobjokey', 'kock', 'kondum', 'kondums',
-  'kum', 'kummer', 'kumming', 'kums', 'kunilingus', 'l3i+ch', 'l3itch', 'labia', 'lmfao', 'lust',
-  'lusting', 'm0f0', 'm0fo', 'm45terbate', 'ma5terb8', 'ma5terbate', 'masochist', 'master-bate',
-  'masterb8', 'masterbat*', 'masterbat3', 'masterbate', 'masterbation', 'masterbations',
-  'masturbate', 'mo-fo', 'mof0', 'mofo', 'mothafuck', 'mothafucka', 'mothafuckas', 'mothafuckaz',
-  'mothafucked', 'mothafucker', 'mothafuckers', 'mothafuckin', 'mothafucking', 'mothafuckings',
-  'mothafucks', 'mother fucker', 'motherfuck', 'motherfucked', 'motherfucker', 'motherfuckers',
-  'motherfuckin', 'motherfucking', 'motherfuckings', 'motherfuckka', 'motherfucks', 'muff', 'mutha',
-  'muthafecker', 'muthafuckker', 'muther', 'mutherfucker', 'n1gga', 'n1gger', 'nazi', 'nigg3r',
-  'nigg4h', 'nigga', 'niggah', 'niggas', 'niggaz', 'nigger', 'niggers', 'nob', 'nob jokey', 'nobhead',
-  'nobjocky', 'nobjokey', 'numbnuts', 'nutsack', 'orgasim', 'orgasims', 'orgasm', 'orgasms', 'p0rn',
-  'pawn', 'pecker', 'penis', 'penisfucker', 'phonesex', 'phuck', 'phuk', 'phuked', 'phuking',
-  'phukked', 'phukking', 'phuks', 'phuq', 'pigfucker', 'pimpis', 'piss', 'pissed', 'pisser',
-  'pissers', 'pisses', 'pissflaps', 'pissin', 'pissing', 'pissoff', 'poop', 'porn', 'porno',
-  'pornography', 'pornos', 'prick', 'pricks', 'pron', 'pube', 'pusse', 'pussi', 'pussies', 'pussy',
-  'pussys', 'rectum', 'retard', 'rimjaw', 'rimming', 's hit', 's.o.b.', 'sadist', 'schlong',
-  'screwing', 'scroat', 'scrote', 'scrotum', 'semen', 'sex', 'sh!+', 'sh!t', 'sh1t', 'shag', 'shagger',
-  'shaggin', 'shagging', 'shemale', 'shi+', 'shit', 'shitdick', 'shite', 'shited', 'shitey', 'shitfuck',
-  'shitfull', 'shithead', 'shiting', 'shitings', 'shits', 'shitted', 'shitter', 'shitters', 'shitting',
-  'shittings', 'shitty', 'skank', 'slut', 'sluts', 'smegma', 'smut', 'snatch', 'son-of-a-bitch'
+  // #
+  '4r5e', '5h1t', '5hit', 'God', 
+  
+  // A
+  'a55', 'a_s_s', 'anal', 'anus', 'ar5e', 'arrse', 'arse', 'ass', 
+  
+  // B
+  'b!tch', 'b00bs', 'b17ch', 'b1tch', 'ballbag', 'balls', 'bastard', 'beastial', 'bellend',
+  'bestial', 'bi+ch', 'biatch', 'bitch', 'bloody', 'blow job', 'blowjob', 'boiolas', 'bollock',
+  'bollok', 'boner', 'boob', 'booobs', 'boooobs', 'booooobs', 'booooooobs', 'breasts', 'buceta',
+  'bugger', 'bum', 'bunny fucker', 'butthole', 'buttmunch', 'buttplug', 
+  
+  // C
+  'c0ck', 'carpet muncher', 'cawk', 'chink', 'cipa', 'cl1t', 'clit', 'cnut', 'cock', 'cok', 'coon',
+  'crap', 'cum', 'cunilingus', 'cunillingus', 'cunnilingus', 'cunt', 'cyalis', 'cyberfuc',
+  
+  // D
+  'd1ck', 'damn', 'dick', 'dildo', 'dink', 'dirsa', 'dlck', 'dog-fucker', 'doggin', 'donkeyribber',
```

Comment:
Shouldn't need dog-fucker b/c we have fuck
```suggestion
  'd1ck', 'damn', 'dick', 'dildo', 'dink', 'dirsa', 'dlck', 'doggin', 'donkeyribber',
```